archive/ar/member_metadata.ksy

```diff
@@ -0,0 +1,22 @@
+meta:
+  id: member_metadata
+  title: Unix ar archive member metadata
+  license: CC0-1.0
```

Comment:
Shouldn't it import `space_padded_number` too?

Reply:
Yup, it should. Why does this even compile? This seems like a bug...